doc/user/layouts/partials/create-source/connector/kinesis/details.html

```diff
@@ -8,3 +8,8 @@
   - The IAM account whose credentials you provide requires
     `kinesis-read` permissions and access to `ListStreams` and `Read`.
 - Kinesis sources will only have one column, which will be named `data`.
+- The following Kinesis features are not yet supported:
```

Comment:
```suggestion
- The following features are not yet supported:
```

Reply:
This mixes Kinesis features (resharding, fan-out) with Materialize features (BYO consistency, "start offsets" WITH option), so maybe it'd be more correct to drop "Kinesis".